tests/EndToEnd/RenderPoints/render.html.twig

```diff
@@ -5,3 +5,18 @@
 {% endfor %}
 
 {{ error }}
+
+{#
+    Twigstan compile this to if (isset($optional) && !(\null === $optional)) {
+    instead of if (array_key_exists($optional, $context) && !(\null === $optional)) {
+#}
+{% if optional is defined and optional is not null %}
```

Comment:
There is false positive error here

Reply:
I found the problem. It was interesting to solve this. 
Fixed in https://github.com/twigstan/twigstan/pull/40